### PR TITLE
Add gcovr and lcov packages

### PR DESCRIPTION
**Pull request by niontive**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [ ] The toolchain has been rebuilt successfully (or no changes were made to it)
- [ ] The toolchain/worker package manifests are up-to-date
- [ ] Any updated packages successfully build (or no packages were changed)
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [ ] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [ ] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [ ] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS/gcovr/gcovr.spec [hunk 1]
```diff
@@ -0,0 +1,125 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+Name:           gcovr
+Version:        5.0
+Release:        2%{?dist}
+Summary:        A code coverage report generator using GNU gcov
+License:        BSD
```
Comment (PawelWMS) [inline SPECS/gcovr/gcovr.spec:7]: Missing `Vendor` and `Distribution` tags.

File: SPECS/gcovr/gcovr.spec [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+%files
+%license LICENSE.txt
+%doc README.rst CHANGELOG.rst
+%{_bindir}/gcovr
+%{python3_sitelib}/gcovr*
+
+
+%changelog
+* Wed Jul 21 2021 Nicolas Ontiveros <niontive@microsoft.com> - 5.0-2
+- Initial CBL-Mariner import from Fedora 35 (license: BSD).
```
Comment (PawelWMS) [inline SPECS/gcovr/gcovr.spec:50]: Please always verify the license and mention that in the changelog.

File: SPECS/lcov/lcov.spec [hunk 3]
```diff
@@ -0,0 +1,144 @@
+Name: lcov
+Version: 1.14
+Release: 6%{?dist}
+Summary: LTP GCOV extension code coverage tool
```
Comment (PawelWMS) [inline SPECS/lcov/lcov.spec:4]: Same comment about `Vendor` and `Distribution`.

File: SPECS/lcov/lcov.spec [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+License: GPLv2+
+
+URL: https://github.com/linux-test-project/lcov/
```
Comment (PawelWMS) [inline SPECS/lcov/lcov.spec:7]: Please lint all new specs - it'll put the tags in the right order, fix paths, etc.

File: SPECS/lcov/lcov.spec [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+HTML output and support for large projects.
+
+%prep
+%autosetup -S git_am
+
+%install
+make install DESTDIR=$RPM_BUILD_ROOT BIN_DIR=%{_bindir} MAN_DIR=%{_mandir} CFG_DIR=%{_sysconfdir}
+
+%files
+%{_bindir}/*
```
Comment (PawelWMS) [inline SPECS/lcov/lcov.spec:35]: Missing `%license` macro.

File: SPECS/lcov/lcov.spec [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+%{_mandir}/man1/*
+%{_mandir}/man5/*
+%config(noreplace) %attr(0644,root,root) %{_sysconfdir}/lcovrc
+
+%changelog
+* Wed Jul 21 2021 Nicolas Ontiveros <niontive@microsoft.com> - 1.14-6
+- Initial CBL-Mariner import from Fedora 35 (license: GPLv2+).
```
Comment (PawelWMS) [inline SPECS/lcov/lcov.spec:42]: Same comment about the license.

File: SPECS/lcov/lcov.spec [hunk 7]
```diff
@@ -0,0 +1,144 @@
+Name: lcov
```
Comment (PawelWMS) [inline SPECS/lcov/lcov.spec:1]: Don't forget about updating the `licenses.json`, regenerating the licenses table, and updating `cgmanifest,json`.

To regenerate the licenses table:

```bash
python3 toolkit/scripts/licenses/license_map.py --only_update \
          SPECS/LICENSES-AND-NOTICES/data/licenses.json \
          SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md \
          SPECS \
          SPECS-SIGNED
```

**Review by PawelWMS — CHANGES_REQUESTED** [review]

**Comment by jslobodzian** [comment]
Closing stale PR.  Gcovr has recently been added to 1.0 and is already in 2.0.

**Pull request closed without merging**